### PR TITLE
Python Add Pasting to setitem operator

### PR DESCRIPTION
**Pull request by blowekamp**

### CODE REVIEW

**Review by zivy — APPROVED** [review]

File: Wrapping/Python/Python.i [hunk 1]
```diff
@@ -577,9 +585,17 @@
 
 
         def __setitem__( self, idx, value ):
-            """Sets the pixel value at index idx to value.
+            """Sets pixel value(s) values.
```
Comment (dave3d) [inline Wrapping/Python/Python.i:588]: what does 'value(s) values' mean?  is value supposed to be there twice?

File: Wrapping/Python/Python.i [hunk 2]
```diff
@@ -577,9 +585,17 @@
...
+
+            The dimension of idx should match that of the image.
 
-            The dimension of idx should match that of the image."""
+            If all indices are intengers then value should be a pixel value
```
Comment (dave3d) [inline Wrapping/Python/Python.i:592]: 'integers'

File: Wrapping/Python/Python.i [hunk 3]
```diff
@@ -577,9 +585,17 @@
...
+            ( scalar or sequence for vector pixels). Value is assigned pixel.
+
+            If all indices are slice indices then, value should be an image
+            of matching size, dimension and type as the image and region. This
+            image's pixel and then assigned the value image with the paste filter.
```
Comment (dave3d) [inline Wrapping/Python/Python.i:597]: do you mean 'are' instead of 'and'?

File: Wrapping/Python/Python.i (hunk 1 above)
Comment (blowekamp) [reply Wrapping/Python/Python.i]: I renamed the argument to rvalue.

**Pull request merged**